internal/models/models.go

```diff
@@ -13,7 +13,9 @@ type Log struct {
 
 type Publisher struct {
 	gorm.Model
-	Name string `json:"name"`
+	OrganizationID string `json:"organizationId"`
+	URL            string `json:"url"`
```

Comment:
This should be a list of URLs (possibly with an additional table and a 1-N relationship): every publisher can have one or more associated GitLab groups (for example) plus different single repos, and so on.